tests/manage/test_remove_mon_from_cluster.py

```diff
@@ -72,6 +71,7 @@ def test_remove_mon_pod_from_cluster(self):
         new mon pod on its own
 
         """
+        HEALTH = CephCluster()
```

Comment:
`verify_mon_pod_up` depends on this variable. We should create a class attribute for the cluster (and pod) where these methods can reference using `self.` instead of a global var. This will also involve moving the methods that depend on those class attributes into the test class itself.

Reply:
for now I just fixed the deployment case to work, once that is successful, I will test this out, I have added "Needs Testing" Label before.